tests/typings/sync-cucumber/cucumber.ts

```diff
@@ -3,6 +3,7 @@ const hook: WebdriverIO.HookFunctions = {
     beforeFeature: function (uri, feature, scenarios) {
     },
     beforeScenario: function (uri, feature, scenario, sourceLocation) {
+        console.log(sourceLocation.line)
```

Comment:
`console.log` ?

Reply:
it is actually a test that `.line` is accessible but probably not that clear like this, will add a comment